style.css

```diff
@@ -1,554 +1,1111 @@
+/* styling the project or work section */
+
+.work-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 386px;
+  padding: 10% 24px 35% 24px;
+  height: auto;
+  gap: 2%;
 }
 
-.work > .first-box > .imgsection > .ttwork > h3 {
-  font-family: 'Crete Round', serif;
-  font-weight: 300;
-  font-size: 2rem;
+.work-container > div {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 400px;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("worksection/work-image.png");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  animation-name: slidein;
+  animation-duration: 2s;
+  animation-delay: 0.5s;
 }
 
-.about > p {
-  font-family: 'Inter', sans-serif;
-  font-size: 16px;
+.work-container > div h3 {
+  font-size: 32px;
   font-weight: 400;
-  line-height: 24px;
-  letter-spacing: 0;
-  text-align: left;
+  margin-top: 74px;
+  line-height: 3rem;
+  font-family: 'Crete Round', serif;
+  height: 81px;
+  width: 100%;
+  color: #fff;
+  padding: 0 16px 0 20px;
 }
 
-.work > .carttt > .cart > p {
-  font-family: 'Inter', sans-serif;
+.work-container > div p {
   font-size: 16px;
   font-weight: 400;
-  line-height: 24px;
-  padding: 0 1rem;
-  letter-spacing: 0;
-  text-align: left;
-}
-
-.work > .first-box > .imgsection > .ttwork > p {
   font-family: 'Inter', sans-serif;
-  padding-top: 0.5rem;
-  margin-right: 1.98rem;
-  font-weight: 400;
+  height: 90px;
+  width: 100%;
+  padding: 0 35px 0 20px;
+  color: #fff;
   line-height: 1.5rem;
-  max-width: 20rem;
-  font-size: 1rem;
-  letter-spacing: 0;
-  text-align: left;
-  flex: none;
-  font-style: normal;
-  flex-grow: 0;
+  margin-top: 24px;
 }
 
-.about > .listing > ul {
+.work-container > div ul {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
+  width: 100%;
+  padding: 0 16px 0 20px;
+  justify-content: flex-start;
+  margin-top: 24px;
+  gap: 15px;
+}
+
+.work-container > div li {
   list-style-type: none;
+  padding: 10px;
+  color: #fff;
+  font-size: 15px;
+  font-weight: 500;
   font-family: 'Inter', sans-serif;
-  line-height: 3rem;
-  margin-bottom: 3rem;
+  background-color: rgba(255, 255, 255, 0.24);
 }
 
-.work > .carttt > .cart > ul {
-  list-style-type: none;
-  display: flex;
-  flex-direction: row;
-  padding: 0 1rem;
+.work-container > div button {
+  height: 50px;
+  width: 100%;
+  text-align: center;
+  line-height: 3rem;
+  font-size: 17px;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+  background-color: #ff6b00;
+  color: #fff;
+  text-decoration: none;
+  margin-top: 22px;
+  border: 0;
 }
 
-.work > .first-box > .imgsection > .ttwork > ul {
-  list-style-type: none;
+/* styling the about section */
+
+.about-container {
   display: flex;
-  flex-direction: row;
-  padding: 1rem 0;
+  flex-direction: column;
+  height: auto;
+  margin: 64% 0 0 0;
```

Comment:
- [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.
![image](https://user-images.githubusercontent.com/58642949/187038727-bff57ff9-8391-440d-bb63-405545bf38db.png)
![image](https://user-images.githubusercontent.com/58642949/187038766-76863bfa-1f77-49d2-ba03-821eecf526bb.png)